logger/logger.go

```diff
@@ -56,8 +61,9 @@ func ContextLog(ctx Valuer, err []error, entry *logrus.Entry) *logrus.Entry {
 
 		// Check last, to allow LogFields to overwrite this package's behaviour.
 		// Do not recurse in error causes, the error itself should merge its causes' fields if desired.
```

Comment:
This comment is still true. What this PR changes is the _default_ behaviour of an error wrapper to inherit its cause's `LogFields`, if it has one.